ui/Screen/ProjectRegistration.svelte

```diff
@@ -0,0 +1,88 @@
+<script>
+  import { Flex, Title } from "../DesignSystem/Primitive";
+  import { ModalLayout, Remote, StepCounter } from "../DesignSystem/Component";
+
+  import RegistrationDetailsStep from "./ProjectRegistration/RegistrationDetailsStep.svelte";
+  import TransactionSummaryStep from "./ProjectRegistration/TransactionSummaryStep.svelte";
+
+  import { projects as projectStore } from "../src/project.ts";
+  import { session as sessionStore } from "../src/session.ts";
+  import { orgMocks } from "../lib/orgMocks.js";
```

Comment:
Can't we replace this, now that the endpoints landed in master?

Reply:
I'll do it in a separate PR. Besides, I don't think we have an endpoint for listing all orgs that I'm a member of, which we need for the dropdown.